src/main/java/io/jenkins/plugins/sprp/PipelineSnippetGenerator.java

```diff
@@ -165,12 +165,12 @@ public String getStage(
                 if (stage.getName().equals("Tests")) {
                     if(testResultPaths != null)
                         snippet += "\t\t\t" + addTabs(getPublishReportSnippet(testResultPaths), 3);
-                    if(gitConfig.getGitUrl() != null)
-                        snippet += "\t\t\t" + addTabs("gitPush " +
-                                "credentialId: \"" + gitConfig.getCredentialsId() + "\"," +
-                                "url: \"" + gitConfig.getGitUrl() + "\"," +
-                                "branch: \"" + gitConfig.getGitBranch() + "\"" +
-                                "\n", 3);
+//                    if(gitConfig.getGitUrl() != null)
+//                        snippet += "\t\t\t" + addTabs("gitPush " +
+//                                "credentialId: \"" + gitConfig.getCredentialsId() + "\"," +
+//                                "url: \"" + gitConfig.getGitUrl() + "\"," +
+//                                "branch: \"" + gitConfig.getGitBranch() + "\"" +
+//                                "\n", 3);
```

Comment:
@gautamabhishek46, every time you comment out code, you **MUST** say what future action is needed. Should this code be deleted? Should it be re-instated? Will it be replaced (and if so when and by who)?

If you comment out code because you don't need it, just delete it. If you keep it around "just in case", delete it: the previous commit has the code.

Reply:
Comment: "Below commented code is to generate gitPush step, this is only commented for testing purpose." has been added.

Reply:
That works for me

Reply:
I understand you comment it out for testing purposes, but you need to put a comment in the code stating that, for example:
```
// TODO Abhishek: code is commented out for testing purposes, it will be (removed|reinstated) later
```